ash/src/entry.rs

```diff
@@ -46,6 +44,22 @@ pub enum LoadingError {
     LibraryLoadError(String),
 }
 
+impl fmt::Display for LoadingError {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "LoadingError::{:?}", self)
+    }
+}
+
+impl Error for LoadingError {
+    fn description(&self) -> &str {
+        "LoadingError"
+    }
+
+    fn cause(&self) -> Option<&Error> {
+        None
+    }
```

Comment:
This method is also deprecated, and this definition is redundant to the default definition anyway.

Reply:
Again, only following precedent with how the other were implemented. I (possibly wrongly?) assumed that smaller changes would go over better than sweeping refactoring.

Reply:
You're not obliged to do sweeping refactoring, but we shouldn't introduce new code using obsolete patterns.

Reply:
Also removed